snaps/cpa-sample-code/README.md

```diff
@@ -0,0 +1,56 @@
+# Snapped CPA Sample Code Command
```

Comment:
Do you think that having only one snap for everything would be easier ? I would call this snap intel-qat and it will contain qatlib examples, qatengine and probably more in the future

Reply:
I think that because these applications are just intended to be references for other snap maintainers/creators that it makes sense to keep them separate for now. But in the future perhaps we should include a reference for a snap with multiple apps/commands.